src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/StoreV3/ContactDiaryStoreSchemaV3.swift

```diff
@@ -0,0 +1,76 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import FMDB
+import CWASQLite
+
+class ContactDiaryStoreSchemaV3: ContactDiarySchemaProtocol {
+
+	// MARK: - Init
+
+	init(databaseQueue: FMDatabaseQueue) {
+		self.databaseQueue = databaseQueue
+	}
+
+	// MARK: - Public
+
+	@discardableResult
+	func create() -> Result<Void, SQLiteErrorCode> {
+		var result: Result<Void, SQLiteErrorCode> = .success(())
+
+		databaseQueue.inDatabase { database in
+			let sql = """
+				CREATE TABLE IF NOT EXISTS ContactPerson (
+					id INTEGER PRIMARY KEY,
+					name TEXT NOT NULL CHECK (LENGTH(name) <= 250),
```

Comment:
Could also use the new constant for maxTextLength

Reply:
I have added a property maxTextLength to the scheme.